relnotes/reverse.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.array.Iteration`
+
+  New module containing single function for now - `reverse`, which allows to
+  iterate given slice in reverse order without any allocations (similar to
+  built-in `foreach_reverse`).
```

Comment:
While not being deprecated in D2, it is considered extremely bad style to use it, and for a good reason - it is a leaky abstraction which wastes a keyword for trivial library utility. We talked about it before in some other issue which I can't currently find :(